.github/workflows/kubernetes-manifests.yml

```diff
@@ -0,0 +1,17 @@
+name: Kubernetes manifests validation
```

Comment:
Simple yet effective validation of Kubernetes manifests. If there's any rendering error, the workflow fails.